recipes/flexiplex/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.97" %}
+
+package:
+  name: flexiplex
+  version: {{ version }}
+
+source:
+  #url: https://github.com/DavidsonGroup/flexiplex/releases/download/Version-{{ version }}/flexiplex-Version-{{ version }}.tar.gz
+  # path: ../
+  url: https://github.com/olliecheng/flexiplex/archive/refs/heads/develop.zip
```

Comment:
Please consider using release as suggested here https://bioconda.github.io/contributor/guidelines.html#stable-urls

Reply:
Hi, thanks for the review. Yes, we will update the recipe once a new release has been made with the revised Makefile, and will mark this as ready for review after that.